private_chaincode_apps/Makefile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Intel Corporation
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+TOP = ..
+include $(TOP)/build.mk
+
+BUILD_DIR := _build
```

Comment:
does it make sense to have a makefile here? As a developer I would assume that each example chaincode (echo, auction,...) is a self-contained project, maybe similar as the SampleCode in the linux-sgx repo. (https://github.com/intel/linux-sgx/tree/master/SampleCode)

Reply:
yes, just to loop through the examples.
the cmake has been removed